src/pytorch_lightning/strategies/launchers/launcher.py

```diff
@@ -0,0 +1,23 @@
+# Copyright The Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from abc import ABC, abstractmethod
+
+from lightning_fabric.strategies.launchers.launcher import _Launcher as _FabricLauncher
+from pytorch_lightning.trainer.connectors.signal_connector import _SIGNUM
+
+
+class _Launcher(_FabricLauncher, ABC):
+    @abstractmethod
+    def kill(self, signum: _SIGNUM) -> None:
+        """Kill existing alive processes."""
```

Comment:
@carmocca You could also push this to fabric and raise NotImplementedError. To avoid having to define a new base class in PL.

Reply:
But do you see us implementing signal handling for fabric in the future? Otherwise I wouldn't do it

Reply:
I think at the core, fabric would expose these primitives yes. You could call `fabric.strategy.launcher.kill()` for example if that's useful to you. The launcher implementation is also very PL-independent. The only reason why they are subclassed in PL is because of some extra state management and hydra support.

Reply:
I'd rather not do this as it would be unused code in Fabric. In the future, we can do that and add docs for it, or try to merge the classes

Reply:
Resolving. We can do this as a follow-up